tools/debug/debugsymbolviewer.py

```diff
@@ -104,7 +104,7 @@ def print_symbol(search_addr):
 		else:
 			l = mid + 1
 
-		mid = (l + h) / 2
+		mid = (l + h) // 2
```

Comment:
Is this right? two slash?

Reply:
Yes this is floor division as per Python 3.x.